build.gradle

```diff
@@ -174,6 +175,13 @@ jacocoTestCoverageVerification {
 // to run coverage verification during the build (and fail when appropriate)
 check.dependsOn jacocoTestCoverageVerification
 
+checkstyle {
+   toolVersion = '8.24'
+   configFile file("checkstyle/checkstyle.xml")
+}
+
+checkstyleMain.ignoreFailures = true
+checkstyleTest.ignoreFailures = true
```

Comment:
Do we want to ignoreFailures?

Reply:
Yes, for now.  The new checkstyle plugin version is ahead of the gradle checktyle, and brings in a lot of changes for `OperatorWrap, CustomImportOrder EmptyLineSeparator` etc. Since I want to keep this change minimal, I am putting `checkstyleMain.ignoreFailures` as true for now.